File: 01.Java-Basic/02.FirstStepsInCodingExercise/src/FishTank.java
import java.util.Scanner;

public class FishTank {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int length = Integer.parseInt(scanner.nextLine());
        int width = Integer.parseInt(scanner.nextLine());
        int height = Integer.parseInt(scanner.nextLine());
        double percent = Double.parseDouble(scanner.nextLine());

        double volume = (length * width * height) * 0.001;
        double usableVolume = volume * (1 - (percent * 0.01));
        System.out.println(usableVolume);
    }
}

File: JavaScript/01.JavaScriptBasic/02FirstStepsExercise/FishTank.js
function solve(input) {
    let length = Number(input[0]);
    let width = Number(input[1]);
    let height = Number(input[2]);
    let percentage = Number(input[3]);

    let volume = (length * width * height) * 0.001;
    volume = volume - (volume * (percentage / 100));
    console.log(volume);
}

solve(["85","75","47","17"]);
solve(["105","77","89","18.5"]);